jupyter_book/commands/__init__.py

```diff
@@ -50,23 +64,61 @@ def main():
 @main.command()
 @click.argument("path-source", type=click.Path(exists=True, file_okay=True))
 @click.option("--path-output", default=None, help="Path to the output artifacts")
-@click.option("--config", default=None, help="Path to the YAML configuration file")
-@click.option("--toc", default=None, help="Path to the Table of Contents YAML file")
+@click.option(
+    "--config",
+    default=None,
+    help="Path to the YAML configuration file (default: PATH_SOURCE/_config.yml)",
+)
+@click.option(
+    "--toc",
+    default=None,
+    help="Path to the Table of Contents YAML file (default: PATH_SOURCE/_toc.yml)",
+)
 @click.option("-W", "--warningiserror", is_flag=True, help="Error on warnings.")
+@click.option(
+    "-n",
+    "--nitpick",
+    is_flag=True,
+    help="Run in nit-picky mode, to generates warnings for all missing references.",
+)
+@click.option(
+    "--keep-going",
+    is_flag=True,
+    help="With -W, do not stop the build on the first warning, "
+    "instead error on build completion",
+)
+@click.option(
+    "--all",
+    "freshenv",
+    is_flag=True,
+    help="Re-build all pages. "
+    "The default is to only re-build pages that are new/changed since the last run.",
+)
 @click.option(
     "--builder",
     default="html",
     help="Which builder to use.",
     type=click.Choice(list(BUILDER_OPTS.keys())),
 )
-def build(path_source, path_output, config, toc, warningiserror, builder):
+def build(
```

Comment:
given these changes to `build` I will hold off updating #687 until this is merged.

Reply:
Yeh thats why I wanted to give you the heads-up! so I don't introduce too many merge conflicts into your PRs lol